sys/Makefile

```diff
@@ -28,6 +28,7 @@ SOURCES_C  = \
 	startup.c \
 	sync.c \
 	sysent.c \
+	sys_mmap.c \
```

Comment:
If `sys_*.c` files are meant to contain syscall implementations I'd like this idea to be applied consistently throught code base.